src/enclave/ccf_v.h

```diff
@@ -40,15 +40,34 @@ extern "C"
 
 #define oe_result_str(x) x
 
+  typedef enum StartType
```

Comment:
There shouldn't be any code duplication with `ccf.edl`. See https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/Edger8rGettingStarted.md#foreign-types